.devcontainer/cpu/devcontainer.json

```
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/anaconda
+{
+  "name": "Caustics - CPU",
+  "build": {
+    "context": "../..",
+    "dockerfile": "../Dockerfile",
+    "args": {
+      "USERNAME": "vscode",
+      "BUILDKIT_INLINE_CACHE": "0",
+      "CLANG_VERSION": ""
+    }
```

```suggestion
    "dockerfile": "../Dockerfile",
    "args": {
      "CLANG_VERSION": ""
    }
```
No need for these args... old configuration for a different image.